cypress/integration/article-view-spec.js

```diff
@@ -0,0 +1,32 @@
+/// <reference types="Cypress" />
+
+import { SearchPage } from '../page-objects/search-page'
+import { ArticlePage } from '../page-objects/article-page'
+
+const searchPage = new SearchPage()
+const articlePage = new ArticlePage()
+
+describe('Article view', () => {
+  beforeEach(() => {
+    cy.navigateToHomePage()
+  })
+
+  it('change article language', () => {
+    searchPage.search('cat')
+    searchPage.results().first()
+    cy.enter()
+    cy.downArrow()
+    cy.enter()
```

Comment:
just another though, do you want to support any one of these? so we don't end up creating a list of typing action.

````
cy.type( 'enter down enter' )
cy.type( [ 'enter', 'down', 'enter' ] )
cy.enter().downArrow().enter()
```

Reply:
just tried it, the third option already valid, i suggest use it.

Reply:
fixed in all the tests